external/storm-hive/src/main/java/org/apache/storm/hive/bolt/HiveBolt.java

```diff
@@ -134,22 +131,16 @@ public void execute(Tuple tuple) {
                     collector.ack(t);
                 tupleBatch.clear();
             }
+        } catch(SerializationError se) {
+            LOG.info("Serialization exception occurred, tuples will NOT be acknowledged ", tuple);
```

Comment:
the message says tuple won't be acked and in next line we ack it, one of the two is not right.